build.gradle.kts

```diff
@@ -258,13 +258,63 @@ configure(subprojectsWithoutToolAndSmokeTestProjects) {
     apply(plugin = "ch.tutteli.gradle.plugins.dokka")
     apply(plugin = "ch.tutteli.gradle.plugins.publish")
 
-    project.extensions.getByType<ch.tutteli.gradle.plugins.publish.PublishPluginExtension>().apply {
+    extensions.getByType<ch.tutteli.gradle.plugins.publish.PublishPluginExtension>().apply {
         resetLicenses("EUPL-1.2")
     }
+
+    val languageSuffixes = setOf("-en_GB", "-de_CH")
+    if (languageSuffixes.any { name.contains(it) }) {
+        extensions.getByType<PublishingExtension>().apply {
```

Comment:
`configure<PublishingExtension> { ...`

Reply:
Basically, whenever you have `.apply {` in `gradle.kts` it means there's something wrong going on.